src/HOHQMesh.jl

```diff
@@ -5,10 +5,14 @@ import HOHQMesh_jll
 export generate_mesh
 
 
+#TODO: add mesh_file_format parsed directly from the control file and set the
+#      appropriate mesh file name extension. Maybe also print a statement saying
+#      which type of mesh you just created
```

Comment:
Is this supposed to be committed permanently or is it just for you?

Reply:
whoops I missed that in my initial cleanup. This was just my TODO list, I will remove it